authorization-for-rag/langgraph-agentic-python/README.md

```diff
@@ -45,7 +45,7 @@ This example demonstrates how to combine [LangChain](https://python.langchain.co
    ```sh
    $ python -m venv venv
    $ source venv/bin/activate
-   $ pip install -r requirements.txt
+   $ PIP_CONFIG_FILE=pip.conf pip install -r requirements.txt
```

Comment:
Was having trouble installing this way on my corporate Mac, simply b/c my pip config list relies on Artifactory registries by default. Thus, have added a `pip.conf` to make this a bit easier on corporate pc's...